src/index.ts

```diff
@@ -1,16 +1,15 @@
-export { Transaction } from './XRP/generated/org/xrpl/rpc/v1/transaction_pb'
-
-export { default as Wallet } from './XRP/wallet'
+// Exported Common Functionality.
```

Comment:
Note I re-ordered this file by:
- Component (alphabetically: Common, PayID, XRP)
- Alphabetically line by line